toolkit/tools/precacher/precacher.go

```diff
@@ -258,7 +290,34 @@ func getPackageRepoPaths(repoUrl string) (packages []string, err error) {
 	return
 }
 
-// downloadMissingPackages will attemp to download each package listed in rpmSnapshot that is not already present in the
+// getPackageRepoUrlsFromRepoFile returns a list of packages available in the given repoUrl by running repoquery
+func getPackageRepoUrlsFromRepoFile() (packageURLs []string, err error) {
+	const (
+		reqoqueryTool    = "repoquery"
+		randomNameLength = 10
+		printErrorOutput = true
+	)
+	// We have removed all other repo files from the chroot, so we can blindly enable all repos to get the full list of packages
+	var queryCommonArgList = []string{"-y", "-q", "--enablerepo=*", "-a", "--location"}
```

Comment:
Do we need `--enablerepo=*`? Won't `repoquery` use all repos by default?

Reply:
I think `enabled=0` in the repofile will disable it by default. I'm assuming that you want to use all the repos you pass in here.